fr.json

```diff
@@ -98,7 +98,7 @@
     "Step 16": "Etape 16",
     "Step 17": "Etape 17",
     "Step 18": "Etape 18",
-    "This method only supports Android 11 and above": "",
+    "This method only supports Android 11 and above": "Cette méthode ne prend en charge qu'Android 11 et supérieur",
     "Got an issue? Check the <a class=\"text-yellow-200 hover:underline\" target=\"_blank\" href=\"https://gist.github.com/Mirai0009/fa4307e97ad40e71f89754c372b3ba91#troubleshooting\">troubleshooting guide here</a> or join our <a class=\"text-yellow-200 hover:underline\" target=\"_blank\" href=\"https://discord.gg/qzz78F9PNr\">Discord server</a>.": "",
```

Comment:
missing translation

Reply:
also in line 75 and 76 i believe

"Sign in with your Google account.": "",
"Sign in with Google": "",